pkg/vmtest/integration.go

```diff
@@ -188,6 +226,12 @@ func QEMUTest(t *testing.T, o *Options) (*qemu.VM, func()) {
 
 	return vm, func() {
 		vm.Close()
+		if !o.NoKernelCoverage {
```

Comment:
can we invert this logic somehow? ! NokernelCoverage gets confusing

Reply:
The default is to collect the coverage if CONFIG_GCOV is enabled, but that can be disabled by setting NoKernelCoverage.